files/default/handler/audit_report.rb

```diff
@@ -124,6 +129,7 @@ def get_opts(format, quiet, attributes)
           'format' => format,
           'output' => output,
           'logger' => Chef::Log, # Use chef-client log level for inspec run,
+          :backend_cache => node['audit']['inspec_backend_cache'],
```

Comment:
Can we change this to use the new hash syntax instead of hash-rockets?

```ruby
backend_cache: node['audit']['inspec_backend_cache']
```

... the stuff above this apparently needs to be strings (but maybe it doesn't, we should probably check on that as some point), but if this key is expected to be a symbol, we should be consistent. 🙂